Not found dist

Error: Cannot find module '/Users/<username>/<project>/node_modules/prisma-offset-pagination/dist/index.js'. Please verify that the package.json has a valid "main" entry

--- a/package.json
+++ b/package.json
@@ -9,7 +9,8 @@
   "scripts": {
     "clean": "rm -rf dist",
     "build": "npm -s run clean && tsc",
-    "tsc": "tsc"
+    "tsc": "tsc",
+    "prepublish": "tsc"
   },
   "devDependencies": {
     "@types/node": "^14.0.22",

